A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A320_Neo_MFD.js

```diff
@@ -859,7 +859,8 @@ class A320_Neo_MFD_NDInfo extends NavSystemElement {
         const ws = this.ndInfo.querySelector("#Wind_Strength");
         const wa = this.ndInfo.querySelector("#Wind_Arrow");
         const wptg = this.ndInfo.querySelector("#Waypoint_Group");
-        if (ADIRSState != 2 || groundSpeed <= 100) {
+        const onGround = Simplane.getIsGrounded();
+        if (ADIRSState != 2 || groundSpeed <= 100 && onGround) {
```

Comment:
**question**: Did you consider using `A:AIRSPEED TRUE`?

Reply:
Negative,  that would mean the TAS and Wind Arrow would become visible as as the Airspeed goes alive right? I did not want to change the original behaviour while takeoff because i dont now if that would be wrong then

Reply:
Not necessarily if you consider TAS above 100kts. However, I don't know what the correct specification is, so do take my comment with a grain of salt.